### PR TITLE
#26 chore: setup redux , action , reducers and store 

### PR DESCRIPTION
**Pull request by PrinceNiyonshuti**
#### What does this PR do?

- Setup Redux

#### Description of Task to be completed?

Redux is set up and created a simple number counter on the landing page using redux as the state management library; action, action creator, and createReducer are implemented.

#### How should this be manually tested?
- Clone the repo

- checkout to branch `ch-setup-redux-26`

- pull the changes by `git pull`
- install all packages by running `npm install`
- start app by running `npm run start`

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

[Trello Board # 26](https://trello.com/c/iNmX95yk/26-setup-redux)

#### Screenshots (if appropriate)

image

#### Questions:

N/A

### CODE REVIEW

**Review by PatrickNiyogitare28 — APPROVED** [review]

**Comment by mukunzidd** [comment]
@PrinceNiyonshuti please follow the best practices for PR titles #card_ref type: Short Description

**Review by Hodal66 — COMMENTED** [review]
Are those buttons work well??

**Comment by PrinceNiyonshuti** [comment]
> Are those buttons work well??

Yes, they work 
but you have to install dependency by running `npm install` first

**Review by PatrickNiyogitare28 — APPROVED** [review]

**Comment by Hodal66** [comment]
Hello, guys let us fix this conflict!!!

**Review by Hodal66 — APPROVED** [review]

**Review by Niyonkuru1 — APPROVED** [review]

**Review by Niyonkuru1 — APPROVED** [review]
Good job

**Comment by tysepa** [comment]
well

File: package.json [hunk 1]
```diff
@@ -23,7 +23,9 @@
     "path": "^0.12.7",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
+    "react-redux": "^7.2.8",
     "react-router-dom": "^6.3.0",
+    "redux": "^4.1.2",
```
Comment (mukunzidd) [inline package.json:29]: Install the latest redux, ignore if this version is the latest.

File: src/index.html [hunk 2]
```diff
@@ -9,6 +9,6 @@
 <body>
     <div id = "root"></div>
 
-    <script src="index.bundle.js"></script>
+    <!-- <script src="index.bundle.js"></script> -->
```
Comment (mukunzidd) [inline src/index.html:13]: Why did you comment this out?

**Review by mukunzidd — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 2 above)
Comment (PrinceNiyonshuti) [reply src/index.html:13]: we commented that line because it was blocking the DOM to be manipulated

File: package.json (hunk 1 above)
Comment (PrinceNiyonshuti) [reply package.json:29]: The lastes version is 4.1.2 which we are using currently

**Review by mukunzidd — APPROVED** [review]

**Pull request merged**